src/src/Controller/Listing/PersonalListingController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller\Listing;
+
+use App\Enum\ListingType;
+use App\Enum\ScopeType;
+use App\Repository\TaskRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class PersonalListingController extends AbstractController
```

Comment:
This is "the same" as WorkListingController - you could use `work` and `personal` as a variable and then have a single route/controller